pkg/entity/data_contract/data_contract.go

```diff
@@ -123,7 +132,7 @@ func list() {
 
 func del(refString *string) {
 	req := &data_contracts.DeleteDataContractRequest{
-		ProjectId: common.ProjectId,
+		ProjectId:       common.ProjectId,
```

Comment:
This should also be the if-else construct.

Reply:
It was merely added to the create functionality as per the ticket

Reply:
now added to all crud actions